api/swagger.yaml

```diff
@@ -243,6 +243,9 @@ definitions:
       ReadOnly:
         description: "Whether the mount should be read-only."
         type: "boolean"
+      Consistency:
+        description: "The consistency requirement for the mount: `default`, `consistent`, `cached`, or `delegated`."
```

Comment:
I think overall the API allows omitting values to use the default, which also is "strictly" what default is 😇;

> In computer technology, a default (noun, pronounced dee-FAWLT ) is a predesigned value or setting that is used by a computer program _when a value or setting is not specified_ by the program user.

What will the result be if an older client creates a mount? (older clients won't know about this option, so use the old API version, and don't specify this property)

@cpuguy83 wdyt?